src/App.css

```diff
@@ -536,11 +548,31 @@ code {
 }
 
 .badge-primary {
-  background-color: #009999 !important;
+  background-color: #099 !important;
 }
 
 .badge-danger {
-  background-color: #ff6666 !important;
+  background-color: #f66 !important;
+}
+
+.app-category {
```

Comment:
I thought I removed these. Did it get back with a merge?

Reply:
I still get an error. Since it is the same color I would keep it that way for now